app/core/Tracker2/LegacyTracker.js

```diff
@@ -0,0 +1,53 @@
+import CocoLegacyTracker from '../Tracker'
+import BaseTracker from './BaseTracker'
+
+/**
+ * Acts as a proxy between our new tracker and our legacy Tracker.coffee
+ * which still handles segment.io and GA tracking.  This tracker also binds
+ * application events to the legacy tracker and exposes legacy tracker methods
+ * externally as necessary.
+ *
+ * TODO remove this tracker when tracker refactor is complete.
+ */
+export default class LegacyTracker extends BaseTracker {
+  constructor (store, cookieConsent) {
```

Comment:
It's a bit wild passing the cookieConsent tracker into this legacyTracker in order to patch the refactored logic. But I can totally get how it maintains the behavior.

When the refactor is completed and this tracker is removed I assume the cookieConsent tracker will stand on its own?

Reply:
Yup and I agree. LegacyTracker was created as a temporary binding between the old tracker and the new tracker and will go away as soon as LegacyTracker goes away (which will be soon - we're just incrementally refactoring the old tracker throughout this project).

Once cookie consent status is used in other trackers we'll either pass it in from the top level tracker or integrate with the vuex store so that other trackers can use the state (probably vuex but TBD).